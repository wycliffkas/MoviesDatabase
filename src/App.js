import React, { Component } from "react";
import Home from "./components/Home";
import MovieDetails from "./components/MovieDetails";
import Cart from "./components/Cart";
import { Provider } from "react-redux";
import store from "./store/store";
import Header from "./commons/Header";
import { Route, BrowserRouter as Router, Switch } from "react-router-dom";

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <Router history={history}>
          <Header />
          <Switch>
            <Route exact path="/" component={Home} />
            <Route exact path="/movie/:id" component={MovieDetails} />
            <Route exact path="/cart" component={Cart} />
          </Switch>
        </Router>
      </Provider>
    );
  }
}

export default App;
