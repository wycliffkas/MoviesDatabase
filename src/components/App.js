import React, { Component } from "react";
// import { exec } from 'child_process';

class App extends Component {
  render() {
    return (
      <div>
        <h1> My React App</h1>
      </div>
    );
  }
}

export default App;
