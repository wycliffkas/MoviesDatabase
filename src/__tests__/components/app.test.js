import App from "../../App";
import React from "react";
import { shallow } from "enzyme";

describe("App Tests", () => {
  const wrapper = shallow(<App />);
  it("test app main page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
