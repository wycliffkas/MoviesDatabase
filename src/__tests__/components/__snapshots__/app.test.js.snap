// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Tests test app main page 1`] = `ShallowWrapper {}`;
